src/views/home/PlayerHome/components/activities/NewsActivity.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <SmartLink :link="link">
+    <div class="page-content card">
+      <div class="container">
+        <div class="row justify-content-between">
+          <div class="col p-0">
+            <!-- Note: Any space between these tags will misalign the Type tag. -->
+            <div :style="{ color: typeColor(article) }">
+              <b>{{ formattedType(article) }}</b>
+            </div>
+          </div>
+          <div class="col p-0" style="text-align:right">
+            <div style="opacity: 0.5; font-weight: bold;">
+              {{ date }}
+            </div>
+          </div>
+        </div>
+
+        <h3 class="card-title" v-if="article.title">{{ article.title }}</h3>
```

Comment:
Adding a `row` class to the `card-title` element (and changing the `margin` rule in the `card-title` class) will align the title to the type header above.